File: 0191-number-of-1-bits/0191-number-of-1-bits.java
class Solution {
    public int hammingWeight(int n) {
        String binRes = Integer.toBinaryString(n);
        int count = 0;
        for(int i=0;i<binRes.length();i++) {
            if(binRes.charAt(i) == '1')
                count+=1;
        }
        return count;
    }
}

File: 0191-number-of-1-bits/0191-number-of-1-bits.js
/**
 * @param {number} n
 * @return {number}
 */
var hammingWeight = function(n) {
    let res = n.toString(2)
    let count = 0;
    for(let i of res) {
        (i==1) && count++;
    }
    return count;
};